homeassistant/components/pyload/sensor.py

```diff
@@ -68,14 +69,20 @@ class PyLoadSensorEntity(StrEnum):
     SensorEntityDescription(
         key=PyLoadSensorEntity.ACTIVE,
         translation_key=PyLoadSensorEntity.ACTIVE,
+        native_unit_of_measurement="downloads",
```

Comment:
Perhaps define this as a constant to avoid repetition? Looks otherwise straightforward :-)